ui/components/Footer/Footer.styles.js

```diff
@@ -0,0 +1,26 @@
+import makeStyles from "@mui/styles/makeStyles";

```

Comment:
@sudo-NithishKarthik this is the style sheet, would i have to use makeStyles here?

Reply:
There are some benefits of using `styled` API over `makeStyles` and definitely material ui making it as their primary means of styling in v5 assures that this is the direction where people are investing in. So, its better we adopt that early on. They way you would achieve it here is, rather than creating a component and applying styles via classes, just create the components and apply styles using `styled` api.

for instance, rather than creating styles for footer, create a footer component using styled api with the necessary styles applied and export it, which then can be used in the file where it will be rendered.

Reply:
Ahh okay, got you. Then I'll change the same for the header component too.

Reply:
Yes, Please